doc/tags.yml

```
@@ -28,159 +29,151 @@ mapping_topics:
-  - nrf/device_guides/nrf53/index.html: ["nrf53-series", "nrf5340", "thingy53", "development-kits",
-                                         "prototyping-platforms"]
-  - nrf/device_guides/nrf53/*.html: ["nrf53-series"]
-  - nrf/device_guides/nrf53/features_nrf53.html: ["nrf5340", "thingy53", "development-kits",
-                                                  "prototyping-platforms"]
-  - nrf/device_guides/nrf53/building_nrf53.html: ["nrf5340", "thingy53", "development-kits",
-                                                  "prototyping-platforms", "nrf-connect-vsc"]
-  - nrf/device_guides/nrf53/fota_update_nrf5340.html: ["nrf5340", "development-kits"]
-  - nrf/device_guides/nrf53/multi_image_nrf5340.html: ["nrf5340", "development-kits"]
-  - nrf/device_guides/nrf53/simultaneous_multi_image_dfu_nrf5340.html: ["nrf5340",
-                                                                        "development-kits"]
-  - /nrf/device_guides/nrf53/serial_recovery.html: ["nrf5340", "development-kits"]
-  - /nrf/device_guides/nrf53/logging_nrf5340.html: ["nrf5340", "development-kits"]
-  - nrf/device_guides/nrf53/thingy53_application_guide.html: ["thingy53", "prototyping-platforms"]
-  - nrf/device_guides/nrf53/qspi_xip_guide_nrf5340.html: ["nrf5340", "development-kits"]
-  - nrf/device_guides/nrf52/index.html: ["nrf52-series", "nrf52840", "nrf52833", "nrf52832",
-                                         "nrf52820",
-                                         "nrf52811", "nrf52810", "nrf52805", "development-kits"]
-  - nrf/device_guides/nrf52/*.html: ["nrf52-series", "development-kits", "nrf52840", "nrf52833",
-                                     "nrf52832", "nrf52820", "nrf52811", "nrf52810", "nrf52805"]
-  - nrf/device_guides/nrf52/features.html: ["protocols", "ble", "blemesh", "esb", "gazell",
-                                            "matter", "multiprotocol", "nfc", "thread", "zigbee"]
-  - nrf/device_guides/nrf52/building.html: ["nrf-connect-vsc"]
-  - nrf/device_guides/pmic/index.html: ["pmic", "npm1100", "npm1300", "npm6001"]
-  - nrf/device_guides/pmic/*.html: ["pmic", "npm1100", "npm1300", "npm6001"]
-  - nrf/device_guides/fem/index.html: ["fem", "nrf21540", "evaluation-kits", "development-kits"]
-  - nrf/device_guides/fem/*.html: ["fem", "nrf21540", "evaluation-kits", "development-kits"]
-  - nrf/device_guides/wifi_coex.html: ["kconfig", "ble", "nrf70-series", "nrf7002", "nrf7001",
-                                       "nrf7000", "wifi"]
+                                                        "nrf9151", "nrf9131", "lte"]
   - nrf/security/security.html: ["kconfig"]
   - nrf/security/ap_protect.html: ["kconfig"]
   - nrf/protocols/*.html: ["protocols"]
   - nrf/protocols.html: ["sidewalk", "ble", "blemesh", "esb", "gazell", "matter", "multiprotocol",
-                         "nfc", "thread", "zigbee", "wifi", "dect-nr+"]
+                         "nfc", "thread", "zigbee", "wifi", "dect-nr+", "protocols"]
```

Protocols is already added to this page through the line above.
```suggestion
                         "nfc", "thread", "zigbee", "wifi", "dect-nr+"]
```

---

The above line is for the files under the protocols folder. This page is outside the protocol's folder.